app/notify/notify_email/school_request_confirmation.dd4490f8-1d7b-455b-8502-47f57a65179a.md

```diff
@@ -1,6 +1,6 @@
 Dear ((school_admin_name)),
 
-((candidate_name)) has sent you a request for school experience from ((placement_start_date)) to ((placement_finish_date)) at ((school_name)).
```

Comment:
Just a reminder, you'll need to update the templates in `Notify` once this gets merged